leaflet-geoman.d.ts

```diff
@@ -1,340 +1,811 @@
+
+            /** The position of a block (draw, edit, custom, options⭐) in the Toolbar can be changed. If not set, the value from position of the Toolbar is taken. */
+            setBlockPosition(block: 'draw' | 'edit' | 'custom' | 'options', position: ControlPosition): void;
+
+            /** Returns a Object with the positions for all blocks */
+            getBlockPositions(): BlockPositions;
+
+            /** To add a custom Control to the Toolbar */
+            createCustomControl(options: CustomControlOptions): void;
+
+            /** Creates a copy of a draw Control. Returns the drawInstance and the control. */
+            copyDrawControl(copyInstance: string, options?: CustomControlOptions): void;
+
+            /** Change the actions of an existing button. */
+            changeActionsOfControl(name: string, actions: (ACTION_NAMES | Action)[]): void;
+
+            /** disable button by control name */
+            setButtonDisabled(name: TOOLBAR_CONTROL_ORDER, state: boolean): void;
+        }
+
+        interface Button {
+            /** actions */
+            actions: (ACTION_NAMES | Action)[];
+
+            /** Function fired after clicking the control. */
+            afterClick: () => void;
+
+            /** CSS class with the Icon. */
+            className: string;
+
+            /**  */
+            disableOtherButtons: boolean;
+
+            doToggle: boolean;
+
+            jsClass: string;
+
+            /** Function fired when clicking the control. */
+            onClick: () => void;
+
+            position: L.ControlPosition;
+
+            title: string;
+
+            toggleStatus: boolean;
+
+            /** block of the control. 'options' is ⭐ only. */
+            block?: 'draw' | 'edit' | 'custom' | 'options';
+
+            /** tool. */
+            tool?: any;
+        }
+
+        interface CustomControlOptions {
+            /** Name of the control */
+            name: string;
+
+            /** block of the control. 'options' is ⭐ only. */
+            block?: 'draw' | 'edit' | 'custom' | 'options';
+
+            /** Text showing when you hover the control. */
+            title?: string;
+
+            /** CSS class with the Icon. */
+            className?: string;
+
+            /** Function fired when clicking the control. */
+            onClick?: () => void;
+
+            /** Function fired after clicking the control. */
+            afterClick?: () => void;
+
+            /** actions */
+            actions?: (ACTION_NAMES | Action)[];
+
+            /** Control can be toggled. */
+            toggle?: boolean;
+
+            /** Control is disabled. */
+            disabled?: boolean;
         }
 
-        export class MapDrawOptions {
+        interface GlobalOptions extends DrawModeOptions, EditModeOptions {
+            /** add the created layers to a layergroup instead to the map. */
+            layerGroup?: L.Map | L.LayerGroup;
```

Comment:
`snappingOrder`, `layerGroup` and `panes` are only on GlobalOptions available
https://github.com/geoman-io/leaflet-geoman#options